usaspending_api/awards/models/subaward.py

```diff
@@ -54,7 +54,7 @@ class Subaward(models.Model):
     )
     unique_award_key = models.TextField(blank=True, null=True, db_index=True)
 
-    latest_transaction_id = models.IntegerField(null=True, blank=True)
+    latest_transaction_id = models.BigIntegerField(null=True, blank=True)
```

Comment:
Are all the other ids Big Ints? Or just this one for some reason?

Reply:
> Award and Transaction IDs are bigints not 32 bit ints
As far as I know, it's just award and transaction ids.

Reply:
Is there a reason for that? Those two are linked to the other data structures.

Reply:
Not sure I understand the question.  The primary key for awards and transactions are big ints so foreign keys that point to those should also be big ints.  The reason awards and transactions are big ints is because they contain millions of records.  Shouldn't be too much longer before they break the 2 billion number afforded by 32 bit integers.